github_jira_sync_app/main.py

```diff
@@ -166,6 +166,15 @@ async def bot(request: Request, payload: dict = Body(...)):
         if payload["action"] not in ["opened", "edited", "closed", "reopened", "labeled"]:
             return {"msg": f"Action was triggered by Issue {payload['action']}. Ignoring."}
 
+        if payload["action"] == "opened":
+            if payload["issue"].get("labels", []):
+                return {
+                    "msg": (
+                        "Action was triggered by Issue Opened with Labels. "
```

Comment:
I think a typo?
```suggestion
                        "Action was triggered by Issue Opened without Labels. "
```

Reply:
`if payload["issue"].get("labels", []):` if the list is not empty, then we process. In case if key does not exist, we assign default value of `[]`

Reply:
Yep sorry I misunderstood the intent here!